stakater/postgres-instance/templates/postgrescluster.yaml

```diff
@@ -26,9 +26,15 @@ spec:
             resources:
               requests:
                 storage: {{ .Values.postgres.repo.storage }}
+#--------Added to reduce WAL file size and WAL file count-----------#
```

Comment:
Why hard code in the template directly?
They are not configurable in values.yaml?

Reply:
I tried lint, it was giving some errors. Will try again and push it after that again.

Reply:
Fixed that. It works now